packages/mattermost-redux/src/actions/posts.ts

```diff
@@ -719,7 +719,7 @@ export function getPostThread(rootId: string, fetchThreads = true) {
         let posts;
         try {
             posts = await Client4.getPostThread(rootId, fetchThreads, collapsedThreadsEnabled);
-            getProfilesAndStatusesForPosts(posts.posts, dispatch, getState);
+            await getProfilesAndStatusesForPosts(posts.posts, dispatch, getState);
```

Comment:
Why change all these to awaits? That'll slow down loading posts since we'll have to wait for the followup requests

Reply:
my local instance was running into errors when I had them as normal invocations. Thinking about it that could also be because i tested with throttled network speed. Let me check if that still happens when not using 3G! :D

Reply:
yep ... that was due to the throttling ... reverted it!

Reply:
That's interesting and mildly concerning. That might be something we need to look into in the future since I could see that applying to people who have poorer connections, but hopefully something like the data manager might help that since we're not reliant on us needing to always load profiles and statuses for all posts right away

Reply:
Keep in mind that I tested with extremely large threads (50K replies), so that might not be the case for the normal usage